tools/codegen/generate.bzl

```diff
@@ -52,9 +52,9 @@ def _gen_template_and_handler(name, importmap = {}):
        "name": name + "_handler",
        "srcs": [ src_desc ],
        "outs": [ gen_handler, gen_tmpl ],
-       "tools": [ "//tools/codegen/cmd/mixgenproc" ],
+       "tools": [ "@com_github_istio_mixer//tools/codegen/cmd/mixgenproc" ],
```

Comment:
non-blocking: have these been bound?  could we use `//external:mixgenproc` here? or are we avoiding `bind`s when possible?

Reply:
I am trying to avoid bind